torch/nn/modules/channelhuffle.py

```diff
@@ -0,0 +1,49 @@
+from .module import Module
+from .. import functional as F
+
+
+class ChannelShuffle(Module):
```

Comment:
Why is this needed?

Reply:
So pixel_shuffle has a similar nn module, which is what I followed. But many times model are written with like `self.channel_shuffle = nn.ChannelShuffle(groups)`. And then their forward just call `self.channel_shuffle(input)`. This is what I have seen. So this maintains that convention.